ch_backup/clickhouse/disks.py

```diff
@@ -157,12 +157,44 @@ def copy_part(
                     part.name,
                     "",
                 )
-                _copy_dir(source_disk.name, source_path, target_disk.name, target_path)
+                self._copy_dir(
+                    source_disk.name, source_path, target_disk.name, target_path
+                )
                 return
         raise RuntimeError(
             f'Disk "{target_disk.name}" path not found for table `{table.database}`.`{table.name}`'
         )
 
+    def _copy_dir(
+        self, from_disk: str, from_path: str, to_disk: str, to_path: str
+    ) -> None:
+        if self._ch_ctl.ch_version_ge("23.9"):
+            command_args = [
+                "--disk-from",
+                from_disk,
+                "--disk-to",
+                to_disk,
+                from_path,
+                to_path,
+            ]
+        else:
+            command_args = [
+                "--diskFrom",
+                from_disk,
+                "--diskTo",
+                to_disk,
+                from_path,
+                to_path,
+            ]
```

Comment:
Options `--diskFrom` and `--diskTo` were renamed to `--disk-from` and `--disk-to` in ClickHouse version 23.9.